### PR TITLE
03-Session-Handling

### PR DESCRIPTION
**Pull request by pablolvillar**
- Added `03-Session-Handling` sample project.
- ⚠️ **Ideally, #3 and #4 should be merged before this one.**

### CODE REVIEW

**Comment by sebacancinos** [comment]
+1 @ntotten

**Comment by pablolvillar** [comment]
Don't merge. This one gotta be fixed.

**Comment by pablolvillar** [comment]
Ok, now it's ready @ntotten

**Comment by pablolvillar** [comment]
This is +1'd @mpaktiti

**Comment by mpaktiti** [comment]
Hi @pablolvillar, we will get back to you once our mobile engineers review this. Thanks.

^ @chenkie

**Comment by pablolvillar** [comment]
`README.md` pending review @sebacancinos

**Comment by sebacancinos** [comment]
+1 to the readme @chenkie

File: 03-Session-Handling/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Session Handling 
+
+- [Full Tutorial](https://auth0.com/docs/quickstart/native/ios-swift/03-session-handling)
+
+The idea of this sample is showing how to achieve session handling in your application, meaning, how to keep the user logged in despite relaunching the app, how to keep his profile up to date, and how to clean everything up when he performs logout.
+
```
Comment (sebacancinos) [inline 03-Session-Handling/README.md]: "in spite of" or "despite" but not "in despite"

**Comment by chenkie** [comment]
@pablolvillar can you address the conflicts here?

**Comment by pablolvillar** [comment]
Solved

**Pull request merged**